### PR TITLE
Developer Guide

### PR DESCRIPTION
**Pull request by rhuttman**

### CODE REVIEW

File: _docs/actions.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+title: Actions
+subtitle: A guide to OAK's actions
+author: ryanh
```
Comment (irsal) [inline _docs/actions.md:4]: i have your avatar as `ryan`

File: _docs/automation-time-apis.md [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+author: ryanh
+tags: [api, time, triggers]
+---
+
+These APIs allow you to schedule an action to occur at a specific minute, and to cancel the task. All the APIs in this section are prefixed by `Automation-time`. 
+
+
+## Launched
+
+### Cancel a Task
```
Comment (irsal) [inline _docs/automation-time-apis.md:13]: i'd consider re-jigging this by importance over alphabetical

**Review by irsal — APPROVED** [review]
nits, i can fix the folder structure and images

File: _docs/automation-time-apis.md (hunk 2 above)
Comment (rhuttman) [reply _docs/automation-time-apis.md:13]: I was thinking more that the list will continue to grow. So cancel should either be on top or bottom. I am guessing yhis means you think on bottom :)

**Pull request merged**